hack/logcheck.out

```diff
@@ -58,7 +58,6 @@
 /config/helpers/bootstrap.go:273:3: function "V" should not be used, convert to contextual logging
 /config/helpers/bootstrap.go:92:4: function "Infof" should not be used, convert to contextual logging
 /config/system-crds/bootstrap.go:58:4: function "Errorf" should not be used, convert to contextual logging
-/pkg/admission/apibinding/apibinding_admission.go:189:3: function "Errorf" should not be used, convert to contextual logging
```

Comment:
are the changes in this file auto-generated?

Reply:
yes